interfaces/manifest.idl

```diff
@@ -0,0 +1,68 @@
+enum AppBannerPromptOutcome {
+    "accepted",
+    "dismissed"
+};
+[Constructor,
+ Exposed=Window]
+interface BeforeInstallPromptEvent : Event {
+    Promise<PromptResponseObject> prompt();
+};
+partial interface Window {
+    attribute EventHandler onappinstalled;
+    attribute EventHandler onbeforeinstallprompt;
+};
+dictionary WebAppManifest {
```

Comment:
This and everything below it is unused,

Reply:
Shouldn't we make the IDL completed in *.idl file? Which would be convenient for automatically sync when IDL changes in future.

Reply:
Hmm, I don't know, I guess auto-syncing is valuable, but that spec is weird because it contains IDL that is not used for defining actual web-facing APIs, and is instead somewhat informative.

Reply:
Maybe @kenchris / @marcoscaceres could answer your concern. :-)

Reply:
@mdittmer and @lukebjerring will be working on that auto-syncing soon. See above discussion FYI.

Reply:
@Honry, we  should only include the stuff that relates to BeforeInstallPrompt, not the rest of the IDL. I'm having a lot of remorse (maybe even a small panic) about us having used WebIDL to define the format... I fear it's going to cause us problems.

Reply:
@marcoscaceres, I'm OK with it. Done in new commit.